auth_api_key/tests/test_auth_api_key.py

```diff
@@ -43,3 +43,9 @@ def test_cache_invalidation(self):
         )
         with self.assertRaises(ValidationError):
             self.env["auth.api.key"]._retrieve_uid_from_api_key("api_key")
+
+    def test_user_archived(self):
+        demo_user = self.env.ref("base.user_demo")
+        demo_user.active = False
+        with self.assertRaises(ValidationError), self.env.cr.savepoint():
```

Comment:
@TDu question: why do you need a savepoint here?

Reply:
I just used the same code that is used in the previous check above.

Reply:
But the test before works w/out :thinking: 
I might be wrong but I'm pretty sure is useless in both cases in this ctx. 
Anyway, not a blocker.